core/appmodule/service.go

```diff
@@ -1,19 +0,0 @@
-package appmodule
-
-import (
-	"cosmossdk.io/core/intermodule"
-	"cosmossdk.io/core/store"
-)
-
-// Service bundles all the core services provided by a compliant runtime
-// implementation into a single services.
-//
-// NOTE: If new core services are provided after core v1, they shouldn't be added to this
-// interface which would be API breaking, but rather a cosmossdk.io/core/appmodule/v2.Service
-// should be created which extends this Service interface with new services.
-type Service interface {
-	store.KVStoreService
-	store.MemoryStoreService
-	store.TransientStoreService
-	intermodule.Client
-}
```

Comment:
I removed this from core because:
1. we don't have `intermodule.Client` implemented yet
2. bundling all the store keys together means we need to register all these keys for every module and I don't think that's smart

So let's revisit the idea of a bundle service later to see if it makes sense